CHANGES.md

```diff
@@ -151,6 +151,21 @@ To be released.
 
 ### Bug fixes
 
+ -  Fixed a bug where executing `Transaction<T>.Actions` had not been atomic.
+    `Actions` in a `Transaction<T>` now became executed all or nothing at all.
+    [[#1267], [#1275]]
+     -  `Transaction<T>.EvaluateActions()` method became atomic.
+     -  `Transaction<T>.EvaluateActionsGradually()` method had returned
+        the same number of `ActionEvaluation`s to `Transaction<T>.Actions`,
+        but now became to omit the evaluations after the first action throwing
+        an exception.  If no action throws any exception, it still returns
+        the same number of `ActionEvaluation`s to `Transaction<T>.Actions`.
+     -  State-wise, `Transaction<T>`s having any exception-throwing action
+        now do not commit any changes at all to `IStore`.
```

Comment:
What is the meaning of `State` you mean? In my understanding, it should use `IStateStore` instead of `IStore`.

Reply:
I was be blurred about that.  Amended: 1f1ed497c892b56c6802a9d15cdeb4cddd1b98db.